zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -371,24 +350,20 @@ benchmarks! {
     }: { Pallet::<T>::handle_expired_advised_market(&market_id, market)? }
 
     internal_resolve_categorical_reported {
-        // a = total accounts
-        let a in 0..10;
         // b = num. accounts with assets
         let b in 0..10;
         // c = num. asset types
         let c in (T::MinCategories::get().into())..T::MaxCategories::get().into();
 
         let c_u16 = c.saturated_into();
-        let (_, market_id) = setup_resolve_common_categorical_after_dispute::<T>(a, b, c_u16)?;
+        let (_, market_id) = setup_resolve_common_categorical_after_dispute::<T>(b, c_u16)?;
```

Comment:
I would expect a `Reported` market here. I would assume, that the market is `Disputed`. Maybe add a benchmark `verify` check at the end of this benchmark to check for the right status.